enki/exceptions.py

```diff
@@ -26,6 +26,18 @@
 """
 
 
+def check_errors(data):
```

Comment:
Well, I've make it as generic as possible, so we can extend it for future checks. Therefore, I put it in there.